_posts/2019-08-03-release-v0-3-6.md

```diff
@@ -0,0 +1,114 @@
+applications will follow soon—watch this space! In the meantime, `godoc` can be
+used to examine our new API functions.
+
+Please note though that **API functions are not yet finalised and may be subject
+to change** in future versions. Yggdrasil is still alpha-grade software at this
+point so all of the usual warnings apply.
+
+### Platform Support
+
+We enjoy great support from our community in bringing and packaging Yggdrasil on
+new platforms. Since the release of version 0.3.5, the following third-party
+packages have cropped up, and we are very grateful to the maintainers:
+
+- A [new RPM build](https://copr.fedorainfracloud.org/coprs/leisteth/yggdrasil/) for Red Hat, Fedora, CentOS etc.
+- An [AUR package](https://aur.archlinux.org/packages/yggdrasil-git/) for Arch Linux
+- A [Void package](https://github.com/void-linux/void-packages/tree/master/srcpkgs/yggdrasil) for Void Linux
+- A [MacPorts package](https://github.com/macports/macports-ports/blob/master/net/yggdrasil-go/Portfile) for macOS
+
+We expect that any third-party packages which have not yet been updated for
+v0.3.6 will be updated soon!
+
+We are aware of a few outstanding issues with Windows, which are largely related
+to one or two bugs in the [Water](https://github.com/songgao/water) library
+which we use for TUN/TAP support. We are hoping to address these problems with
+the maintainer of this library soon. Using Yggdrasil in router-only mode does
+work as expected, but some bugs when using the TAP adapter still remain. In the
+meantime, we'd certainly welcome any assistance in maintaining the Windows port
+of Yggdrasil.
+
+The iOS build has been largely neglected due to API changes, although hopefully
+a new TestFlight build for version 0.3.6 will be available before too long.
+
+### Upgrading
+
+We recommend that all Yggdrasil users always run the latest version of the code
+wherever possible, so please upgrade as soon as it is convenient. New downloads
+are available from our [Builds](builds.md) page and
+[Neil](https://github.com/neilalexander)'s S3 repositories are up-to-date for
+Debian and EdgeRouter installs.
+
+If you have installed through a package manager, you should be able to upgrade
+in-place as soon as the new packages are available. On macOS, you can simply
+install the new `.pkg` from the builds page over the top of the old one. On
+Windows, and on any installation where the binary was installed by hand, you can
+simply replace the `yggdrasil` and `yggdrasilctl` binaries with the newly
+released builds.
+
+Building from source is simple if you have Git and Go 1.11 or later installed:
+```
+git clone https://github.com/yggdrasil-network/yggdrasil-go
+cd yggdrasil-go
+./build
+```
+
+### Feedback
+
+We always welcome feedback, so please do feel free to join us either in our
+[Matrix channel](https://riot.im/app/#/room/%23yggdrasil:matrix.org) or on IRC
+in `#yggdrasil` on Freenode. You can also raise bug reports and issues in [our
```

Comment:
I would say `[freenode](https://freenode.net/)` as I think they prefer all lowercase even in the middle of sentence, but I guess the letter size doesn't matter that much.